lib/ftp.c

```diff
@@ -3689,8 +3709,14 @@ static CURLcode ftp_do_more(struct Curl_easy *data, int *completep)
         /* otherwise just fall through */
       }
       else {
-        result = ftp_nb_type(data, ftpc, ftp, data->state.prefer_ascii,
-                             FTP_RETR_TYPE);
+        if(data->set.prequote && !ftpc->file) {
+          result = ftp_nb_type(data, ftpc, ftp, true,
```

Comment:
```suggestion
          result = ftp_nb_type(data, ftpc, ftp, TRUE,
```
nit, to match rest of source.